automation/run_ios_tests.sh

```diff
@@ -2,8 +2,7 @@
 
 set -euvx
 
-carthage bootstrap --platform iOS --cache-builds
-
+./megazords/ios-rust/build-xcframework.sh --build-profile release
```

Comment:
This is a bit unfortunate because we actually try to run `build-xcframework.sh` as one of the steps in the XCode, however if the xcframework is not built before we run the tests -- we run into a plethora of linker/cc errors and other related issues. This works for now and I'll create a follow up task to revisit when we call these scripts